ikuseiGmbH/smart-village-app-app
# feature(ar): viro integration

- added package `@viro-community/react-viro`
  to implement
- updated `metro.config.js` for the assets that
  the ar feature will support
- added to `@viro-community/react-viro` plugin
  in `app.json` for expo to support
- added `Pods` for augmented reality
  to work on `iOS` devices

SVA-633

## How to test:

run `expo prebuild --clean --no-install` for a proper installation

Copy the following `.zip` file into `src/screens/augmentedReality`. To see the 3D object:

[ARShowTest.zip](https://github.com/ikuseiGmbH/smart-village-app-app/files/9113490/ARShowTest.zip)

select `Touren und Orte` from the `Drawer Menu` in the application. Enter any tour in the `Touren` section at the bottom of the list. Go to the augmented reality detail page from the list. Download any augmented reality object. You can access the page prepared for augmented reality by pressing the `AR Kunst gucken` button to view the ar object.



* [ ] iOS landscape mode


## Screenshots:


|Android|iOS|
|--|--|
![Screenshot_20220707-235028](https://user-images.githubusercontent.com/11755668/177886080-9ccfdbeb-6147-41bb-9a1e-d69b258db1ba.png) | ![IMG_0061](https://user-images.githubusercontent.com/11755668/177886321-8719f41f-6442-4ae9-b3a4-c31a8856c4ed.png)
|Simple Box Data|Simple Box Data|
